ML-for-S2S/ML-for-Derecho/train-torch-unet-s2s-gust.py

```diff
@@ -0,0 +1,399 @@
+import argparse
+import numpy as np
+import pandas as pd
+import xarray as xr
+from datetime import timedelta
+import matplotlib.pyplot as plt
+import xskillscore as xs
+
+import torch
+import torch.nn.functional as F
+from torch.utils.data import DataLoader
+import torch.nn as nn
+import torch.optim as optim
+
+import torch_funcs
+import torch_s2s_dataset_gust
+
+############################################################################
+############################################################################
+
+parser = argparse.ArgumentParser(description='Training UNET for bias-correcting a field (tas2m/prsfc) from CESM.')
+
+parser.add_argument("--variable",choices=["tas2m", "prsfc"],required=True,type=str, help="Variable to be corrected.")
+parser.add_argument("--week",choices=["1", "2", "3","4","5","6"],required=True,type=str, help="Week (lead time) predicted.")
+parser.add_argument("--data_dir",required=False,default='/glade/gust/scratch/molina/', type=str, help="Data storage location.")
+parser.add_argument("--dates_training",required=False,default="1999-02-01/2015-12-31", type=str, help="Initial and final dates of training subset with the format '1999-02-01/2015-12-31'")
+parser.add_argument("--dates_val",required=False,default="2016-01-01/2017-12-31", type=str, help="Initial and final dates of validation subset with the format '2016-01-01/2017-12-31'")
+parser.add_argument("--dates_test",required=False,default="2018-01-01/2020-12-31", type=str, help="Initial and final dates of test subset with the format '2018-01-01/2020-12-31'")
+parser.add_argument("--region_type",choices=["fixed", "random"],default='fixed',required=False, type=str,help="Region method used. Defaults 'fixed' uses one region. 'random' changes regions.")
+
+parser.add_argument("--lon0",required=True, type=str, help="Bottom left corner of 'fixed' region (0 to 360).")
+parser.add_argument("--lat0",required=True, type=str, help="Bottom left corner of 'fixed' region (-90 to 90).")
+parser.add_argument("--dxdy",required=False, type=str, default="32", help="number of grid cells for 'fixed' or 'random' region. Defaults to 32.")
+
+
+#### HYPERPARAMETERS ####
```

Comment:
I would go ahead and change all these hyperparameters to integers as well except for the learning rate. Thanks @jhayron-perez !